config/core/test-configs.yaml

```diff
@@ -2146,6 +2150,9 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
```

Comment:
There are 7 vim3 in Collabora's lab and I believe a few more in some other labs, so this looks fine. We might even be able to enable another 2 or 3 kselftest plans on this platform, but let's see. Other projects are using them too (Mesa CI).